### ISSUE TITLE
"Remote override Step 7: Common Errors" suggests installing Dev branch of Loop

### ISSUE THREAD

**Issue by bjornoleh** [LoopKit/loopdocs#245]
https://loopkit.github.io/loopdocs/nightscout/remote-overrides/#step-7-common-errors

![image](https://user-images.githubusercontent.com/63544115/131973186-0734a259-9759-4b64-aa4c-45e42ebd56f4.png)

**Comment by marionbarker** [comment]
Thanks for pointing this out.  This page may have other errors than just fixing the reference to dev branch and I could use help identifying them. I've been told the Nightscout / IFTTT page also has some incorrect steps.

I do not use remotes at all, so any help in modifying this page and IFTTT would be appreciated.

Is it sufficient to point people to: 
* https://nightscout.github.io/nightscout/pushover

If so, can the content on these pages be mostly replaced with a link to that page.

**Comment by bjornoleh** [comment]
I am unsure about the status of that Pushover page, I just asked about that in the NS docs. I don't use IFTTT or Pushover myself.

We do use Remote Overrides. The instructions for setting up and using Remote Overrides seem pretty accurate. There are some outdated details about versions of NS etc (https://loopkit.github.io/loopdocs/nightscout/remote-overrides/#step-3-update-nightscout-site), and the "Common errors" part about Loop Dev noted above, but nothing major I think (just glanced quickly over it now).

**Comment by Nightfoxy** [comment]
I agree with @bjornoleh review. Removing DEV is a big deal. I get asked this a lot.

1. Reword step 3 to say that v13.0.1 and new is required, not that its the latest.
2. Either remove the link to the NS update video since its missing the step to disconnect and re-connect your github repo in Heroku after deleting the NS repo, OR add another link next to it in the green box to link to the NS update instructions in the NS docs.

Looks good other than that.

**Comment by Kdisimone** [comment]
https://youtu.be/6qSppvgGxpg

Probably a good time to remind people about how to do pull requests to help fix some of the smaller loopdocs things like typos or old references. Take the load off Marion to find all the stragglers that inevitably come with time and changes

**Comment by marionbarker** [comment]
Thanks to all who are participating. In this case, I started working my way through the NS section in order this morning and have made edits on every page so far (starting from Overview and working my way through).

Happy to have PR as well - but if you aren’t set up to do a PR easily, an Issue is fine.

**Comment by marionbarker** [comment]
I have updated the github workflow action file so that whenever I push an update to the working-docs branch on my personal github fork of loopdocs, my version of the deployed pages gets automatically updated.  If you want, you can follow along with my updates by viewing the red/green diffs displayed here:

https://github.com/marionbarker/loopdocs/compare/master...working-docs

If there are no diffs, then my version of deployed pages should match LoopDocs.  If there are diffs, I'm working on the next PR.

My version of deployed pages is here:

https://marionbarker.github.io/loopdocs

**Comment by marionbarker** [comment]
I have completed my initial rewrite of all pages in the LoopDocs Nightscout section.

Several pages have been essentially replaced with links to the updated NS documentation.
I am uncertain about the pebble page - is this still valid or should it be removed?

I will leave these pages out for review for 24 hours.  At that time, I'll set up a PR and bring them into the LoopDocs unless we are still working on updates, in which case, I'll finish the discussion (of course).

![Screen Shot 2021-09-09 at 5 03 58 PM](https://user-images.githubusercontent.com/19607791/132777964-ff1b7205-de33-48d1-b4ba-fff6fe60c078.png)

If not listed below, the edits were pretty minor.

Revised: 
* Retrofit Existing NS (update_user.md)
* Remote Overrides

Point to Nightscout docs by link but keep the page in the menu.:
* Setup Troubleshooting
* IFTTT
* Remote Notifications (pushover)
* Reports

**Comment by psonnera** [comment]
@marionbarker thanks a lot for pointing this to me. The original https://nightscout.github.io/ came directly from Loopdocs using @aviau's work and has been slowly remodelled. Still, IFTTT remained the original historical page and hasn't been correctly integrated. If you will rely on https://nightscout.github.io/ to support your Nightscout section it really needs to remain up to date. Don't hesitate to wake me up and any suggestion is more than welcome.

**Comment by bjornoleh** [comment]
Thanks Marion, you are amazing! I haven't had the time to review your changes, but I can comment briefly on Pebble:

The "app store" of Pebble is discontinued since 2016. But it is still alive on https://rebble.io/, where further instructions can be found.

We use an old Pebble, and it is great! I have not tried the SkyLoop Predict watchface, but there is another one called **Urchin CGM** that can be installed without messing with the rebble.io stuff, just download from Github and install from the Pebble app (after enabling engineering/development mode if I remember correctly).

This is a fork of Urchin that recently got a small update, the original repo is inactive:
https://github.com/woernsn/urchin-cgm

You can probably leave the Pepple page as is for now, and I might look into it later. Perhaps I'll enter an issue report so I don't forget.

**Comment by bjornoleh** [comment]
Regarding authentication here:
 https://marionbarker.github.io/loopdocs/nightscout/update_user/#authenticate-site

Perhaps mention that tokens can also be used, not only API secret? 

http://nightscout.github.io/nightscout/security/#create-authentication-tokens-for-users

**Comment by bewest** [comment]
Looks good so far.  For the Pebble page, consider pointing at the "wearable" page: https://nightscout.github.io/nightscout/wearable/?

**Comment by marionbarker** [comment]
Thanks @bewest - for the wearables page link.  Kate Farnsworth said that she doesn't have time to rewrite the watchface described on the pebble.md page and asked me to remove it.  

Thanks @bjornoleh - I've added the link to tokens and slightly revised the authenticate section:
* https://marionbarker.github.io/loopdocs/nightscout/update_user/#authenticate-site

Given that the last 4 menu items all point to the Nightscout Documentation, I removed all four and added a single page with Headers (so search will work) and Links to Nightscout docs to replace them.
* https://marionbarker.github.io/loopdocs/nightscout/ns_crossref/

I appreciate all the assistance.  I'll wait a bit longer before doing the pull request/publish to LoopDocs.  Feel free to let me know if there are more updates needed. I'll leave this issue open until I actually publish.
